packages/main/src/MenuItemTemplate.tsx

```diff
@@ -30,9 +30,22 @@ function listItemContent(this: MenuItem) {
 		{this.text && <div class="ui5-menu-item-text">{this.text}</div>}
 
 		{rightContent.call(this)}
+		{checkmarkContent.call(this)}
 	</>);
 }
 
+function checkmarkContent(this: MenuItem) {
+	return !this._markChecked ? "" : (
+		<div class="ui5-menu-item-checked">
+			<Icon
+				part="checked"
```

Comment:
This part is not documented, or is it used only internally?
If you would like to provide it to the outside - it should be documented and probably "checked-icon" | "checkmark" | "checkmark-icon" are more appropriate then "checked" to better reference the given DOM element